src/highlight.js

```diff
@@ -11,7 +11,8 @@ https://highlightjs.org/
 
   // Setup highlight.js for different environments. First is Node.js or
   // CommonJS.
-  if(typeof exports !== 'undefined') {
+  // `nodeType` is checked to ensure that `exports` is not a HTML element.
+  if(typeof exports !== 'undefined' && !exports.nodeType) {
```

Comment:
I wonder if there is a more robust brand check we can do? 

Object.prototype.toString.call(exports).includes("HTML")

Maybe not... not sure if anyone is using this with XML nodes. 🤔

Reply:
I think this is pretty robust (and it's the same as underscorejs is using). In any case where `exports` is defined and has the `nodeType` on the object, it should not be detected as being nodejs or commonJS. What do you mean with XML nodes? Higlighting XML is not affected, as this is only the initialising of the script. Running this inside XML should be fine as well, as that way the nodeJS way of initialising shouldn't be used. Does that answer your question?